source/tests/common/test_pairtab.py

```diff
@@ -0,0 +1,225 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+import unittest
+from unittest.mock import (
```

Comment:
Maybe we should move this file somewhere else. Create a new test folder for dpmodel?

Reply:
> Maybe we should move the `numpy` model tests somewhere else. Create a new test folder for `dpmodel`?

I agree.

Reply:
I will create the dpmodel folder in a new PR.